.gitignore

```
@@ -0,0 +1,8 @@
+**\.class
+*.txt
+\bin\*.*
+/.metadata/
+*\.class
+\bin
+/bin/gui/*.*
+/bin
```

```suggestion
*.txt
bin
.metadata
*.class
```
Sen borde du inte behöva några andra .gitignore filer